docs/README.md

```diff
@@ -13,8 +13,11 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 ## How to
 
 - [How to add root key](howto/add_root_key.md)
-- [How to add pagination links](howto/add_pagination_links.md)
 - [Using AMS Outside Of Controllers](howto/outside_controller_use.md)
+- [How to add pagination links](howto/add_pagination_links.md)
+
+#### Specific to JSON API
+- [Customizing the type](howto/jsonapi/custom_type_value.md)
```

Comment:
#### JSON API Resource `type`

##### Implicit

Resource class name is used, e.g. ....

##### Explicitly

In Serializer, declare `type foo`